src/main/java/com/shaneschulte/mc/clanarena/ClanArenaListener.java

```diff
@@ -0,0 +1,18 @@
+package com.shaneschulte.mc.clanarena;
+
+import com.shaneschulte.mc.clanarena.utils.MsgUtils;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class ClanArenaListener implements Listener {
+
+    @EventHandler
+    public void onChallenge (ChallengeStartEvent event) {
+        if (event.isCancelled()) {
```

Comment:
instead of this, you can do (ignoreCancelled = true) in the @EventHandler annotation